crates/wasm/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "penumbra-wasm"
-version = "0.55.0"
+version = "0.55.6-indexeddb"
```

Comment:
Version should remain `0.55.0`, because we set the version to the same value across all crates, and increment in tandem at release time.